mercury.spec

```diff
@@ -14,7 +14,7 @@ License:  Aregonee National Laboratory, Department of Energy License
 URL:      http://mercury-hpc.github.io/documentation/
 Source0:  https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
 Patch0:   https://github.com/daos-stack/mercury/cpu_usage.patch
-Patch1:   https://github.com/daos-stack/mercury/cxi_provider_plus_daos_9561.patch
+Patch1:   https://github.com/daos-stack/mercury/na_ucx_changes.patch
```

Comment:
This should be Patch2:
not replacement for Patch1

Reply:
Yep, removed it accidentally.  It's back in the mercury.spec as patch 1.  Thanks for catching this.